SalesforceCommerceCloud/pwa-kit
# [Product Tile] Promotion callout message (@W-13974753@)

This PR implements the default behaviour for promotion callout. It relies on Alex's display-price work in her PR #1760 .

If the rendered price is a promotional price, we'll also render that promo's callout message in the ProductTile. Otherwise, if the variant has promotions, we'll show the callout from the very first promotion.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. Navigate to this PLP: [/search?q=roll%20sleeve%20blouse](http://localhost:3000/global/en-GB/search?q=roll%20sleeve%20blouse)
2. Verify the promo callout message to be "10$ Off roll sleeve blouse". 34.16 is a promotional price, and we're rendering the callout message of that promotion.

Also, try other kinds of products:
- [without promotions](http://localhost:3000/global/en-GB/category/mens-accessories-gloves)
- [no variants](http://localhost:3000/global/en-GB/category/mens-accessories-luggage) (for the first 3 results)
- [product set](http://localhost:3000/global/en-GB/category/womens-outfits) 

Also, try other pages:
- homepage
- PDP (and its recommended products)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
